account_invoice_overdue_reminder/data/overdue_reminder_result.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2020 Akretion France (http://www.akretion.com/)
+  @author Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<odoo noupdate="1">
+
+
+<record id="voicemail" model="overdue.reminder.result">
```

Comment:
```suggestion
    <record id="voicemail" model="overdue.reminder.result">
```
Could you tab before code, it'll look good.

Reply:
In v13, I'll apply black formatting. In the meantime, let's focus on the important things please.